pkg/cache/manager.go

```diff
@@ -136,3 +86,24 @@ func (m *Manager) CacheUsage(ctx context.Context, blobID string) (snapshots.Usag
 
 	return usage, nil
 }
+
+func (m *Manager) RemoveBlobCache(blobID string) error {
+	blobCachePath := path.Join(m.cacheDir, blobID)
+	blobCacheSuffixedPath := path.Join(m.cacheDir, blobID+dataFileSuffix)
+	blobChunkMap := path.Join(m.cacheDir, blobID+chunkMapFileSuffix)
+	blobMeta := path.Join(m.cacheDir, blobID+metaFileSuffix)
+
+	stuffs := []string{blobCachePath, blobChunkMap, blobMeta, blobCacheSuffixedPath}
```

Comment:
Should remove `blobChunkMap` first, then `blobCachePath` or `blobCacheSuffixedPath`?

Reply:
Yes. Good catch. Fixed